mkdocs/topics/including.md

```diff
@@ -0,0 +1,53 @@
+# Including Other Files and Projects
+
+## Local Files
+
+A good organizational tool is to break your `visivo_project.yml` into multiple local files based on organizational needs.  An example of this would be to put all your `models` in one file and reference that file like:
+
+```
+includes:
+  - path: models.yml
+```
+
+You may wish to organization it along a different axis, for example one file per dashboard like:
+
+```
+includes:
+  - path: finance_dashboard.yml
+  - path: operation_dashboard.yml
+```
+
+Use whatever system best matches your use case.
```

Comment:
Might be good to touch on how you can nest includes within child files for further flexibility in designing a structure that fits the readers business.